src/centralserver/admin-ui/src/components/ui/InitNotification.vue

```diff
@@ -42,7 +42,7 @@
         color="primary"
         outlined
         data-test="notification-button"
-        @click="continueInit"
+        @click="setContisdfnueInit(false)"
```

Comment:
I think there is some typo. At least incorrect language, if not unknown function?
```suggestion
        @click="setContinueInit(false)"
```